### PR TITLE
added paragraph about the android beta client, to review

### PR DESCRIPTION
**Pull request by lefherz**
Do we want to feature the android beta client on our testpilot page? Discuss!

### CODE REVIEW

**Comment by lefherz** [comment]
@stkjj @guruz @pmaier1 @jnweiger @Kawohl

**Comment by Kawohl** [comment]
could you please provide also the link to staging where this would appear?

**Comment by lefherz** [comment]
of course: https://staging.owncloud.org/install/#testing-development

File: install-instructions.php [hunk 1]
```diff
@@ -389,6 +389,8 @@
 		      <a href="<?php echo $DOWNLOAD_CLIENT_DESKTOP_TEST_SOURCES; ?>" class="btn btn-lg btn-default"><i class="icon-archive"></i> Sources</a>
 		    </div>
 		    <?php } ?>
+                    <h2>Android Beta Client</h2>
```
Comment (pmaier1) [inline install-instructions.php]: Please take care for consistency by adding the entry like the 'Desktop Clients' entry: **Android Client** [Version]

File: install-instructions.php [hunk 2]
```diff
@@ -389,6 +389,8 @@
...
+                    <p>If you want to test the Android Beta Client, you can <a href="https://f-droid.org/packages/com.owncloud.android.beta/">download it on Fdroid.</a></p>
```
Comment (pmaier1) [inline install-instructions.php]: Same here:

"Our mobile team provides regular beta releases. You can install them from the F-Droid Android app store (link). Please report any issues to the tracker (link)."

**Comment by lefherz** [comment]
Alright, I implemented the requested changes.

@davivel is the android beta client on google play, too? And if we put this on the website, could you update the $VERSIONS_CLIENT_ANDROID_TESTING variable every time we publish a new beta?

**Comment by davivel** [comment]
Great! :+1:

**Comment by lefherz** [comment]
Okay, now it is finally merged and works, but it doesn't look very beautiful: https://staging.owncloud.org/install/#testing-development
@stkjj ideas?

**Comment by guruz** [comment]
> but it doesn't look very beautiful: 

Put an FDroid logo there?

Also it should be "Testpilot Desktop Client" not "Testpilot Client"

**Comment by pmaier1** [comment]
> Put an FDroid logo there? Also it should be "Testpilot Desktop Client" not "Testpilot Client"

Both good points. Please adapt, thanks!

**Pull request merged**